applications/OptimizationApplication/custom_python/optimization_python_application.cpp

```diff
@@ -123,11 +123,13 @@ PYBIND11_MODULE(KratosOptimizationApplication, m)
 
     // thickness control
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PT);
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PPT);
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, CT);
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, FT);    
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, D_CT);  
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, D_PT); 
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, D_PT_D_FT); 
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, D_PPT_D_FT); 
```

Comment:
As we discussed, we need to write some explanations of these variables.

Reply:
@sunethwarna I will make a separate PR for making h, cpp for classes and also for giving description of variables. Agreed ?!